src/utils.py

```diff
@@ -51,11 +49,11 @@ def ensure_folder_exists(folder_name: str, current_dir: str) -> None:
         print(f"The '{folder_path}' folder has been created.")
 
 
-def save_issues_to_json_file(issues: List[Issue], object_type: str, output_directory: str, state_name: str) -> str:
+def save_to_json_file(state_to_save: list, object_type: str, output_directory: str, state_name: str) -> str:
```

Comment:
Rename:
- state_to_save to items?
- state_name to context_name

Reply:
Renamed as suggested.